docs/content/usage/issue-pull-request-templates.en-us.md

```diff
@@ -219,6 +232,8 @@ Validations:
 |----------|------------------------------------------------------|----------|---------|---------|--------------|
 | required | Prevents form submission until element is completed. | Optional | Boolean | false   | -            |
 
+visible: Default is **[form, content]**
```

Comment:
Shouldn't this be in one of the above tables? Same for the rest of this document.

Reply:
Nobas the table is only for attributes ... we dont have a table for root elements